src/components/PortalButtonSet/PortalButtonSet.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import './PortalButtonSet.css';
+
+function PortalButtonSet({labels, activeButton = 0, onClick}) {
```

Comment:
Add a prop for determining the location of the box-shadow (top/bottom). If not provided then no box shadow